commercetools-sdk-base/src/main/java/io/sphere/sdk/queries/PagedResult.java

```diff
@@ -50,6 +50,22 @@
      }
 
     /**
+     * calculates the page number of the result.
+     *
+     * @return the page number of the result.
+     */
+    default long getPage() {
+
+        if (getOffset() == null) {
+            throw new UnsupportedOperationException("Can only be used if the offset is known.");
+        }
+        if (getCount() == null || getCount() <= 0) {
```

Comment:
And here it's save to assume that `getCount()` isn't null and that it's always positive.